### ISSUE TITLE
Error when trying to retrieve the entity attributes 

### ISSUE THREAD

**Issue by nazarenkoartem** [conwetlab/ngsijs#13]
Dear Developers,

when implementing connection class https://conwetlab.github.io/ngsijs/stable/NGSI.Connection.html faced with some difficulties, namely by implementing following methods in NGSIv2:

getEntityAttribute

Properties "attribute" and "id" are not marked as required, while they are. By sending the following curl: 

`curl -s -H "Content-Type: application/json" -X POST -d '{"bodyParams": {"options": {"attribute": "brightness", "id": "LED002", "type": "LED"}}}' http://localhost:3000/accessfunctionality/orion/getEntityAttribute`

Get an error: 

`{"status":"error","message":"Cannot create property 'search' on string 'http://xxxxxxxx/v2/entities/LED002/attrs/brightness'"}`

NGSI.Connection:

```javascript
 getEntityAttribute(connection, params){
    return new Promise(function(resolve, reject){
      console.log('NGSI function call: getEntityAttribute');

      connection.v2.getEntityAttribute(params.options).then(
        (response) => {
			resolve(response);
        }, (error) => {
			reject(error);
        }
      );
    });
  }
```

The same difficulties I faced with some other methods:
replaceEntityAttributeValue
replaceEntityAttribute
deleteEntityAttribute
getEntityAttribute

Some testing: 

```javascript
 getEntityAttribute(connection, params){
    return new Promise(function(resolve, reject){
      console.log('NGSI function call: getEntityAttribute');

	  console.log(params);
      connection.v2.getEntityAttribute(params.options).then(
        (response) => {
			console.log("correct");
			resolve(response);
        }, (error) => {
			console.log("error");
			reject(error);
        }
      );
    });
  }
```

Result: 

`-> controller: AccessFunctionality
NGSI connection
NGSI function call: getEntityAttribute
{ options: { attribute: 'brightness', id: 'LED002', type: 'LED' } }
POST /accessfunctionality/orion/getEntityAttribute 200 115.451 ms - 138
Executing (default): INSERT INTO "Performances" ("id","component","name","service","executionTime","status","errorMessage","createdAt") VALUES (DEFAULT,'Enabler','orion','getEntityAttribute',4.087831,'Error','Cannot create property ''search'' on string ''http://xxxxxxxx/v2/entities/LED002/attrs/brightness''','2017-11-07 16:37:06.750 +00:00') RETURNING *;
Executing (default): INSERT INTO "Performances" ("id","component","service","executionTime","status","createdAt") VALUES (DEFAULT,'Enablers Framework','/accessfunctionality/orion/getEntityAttribute',118.999163,'Success','2017-11-07 16:37:06.755 +00:00') RETURNING *;`

I will really appreciate, if you could help me. Thank you in advance!

**Comment by aarranz** [comment]
Thanks for reporting those problems. We have updated the documentation to mark the `id` and `attribute` options required for those methods and we think that we have fixed your problem. I will make a release ASAP.

**Comment by aarranz** [comment]
We have released v1.0.3. Please give it a try and let us know if this version fixes your problems.

**Comment by nazarenkoartem** [comment]
Thank you very much for your quick response and your kind help! It helped a lot!
